pkg/utils/utils.go

```diff
@@ -50,3 +52,13 @@ func SaveToFile(fileName string, content []byte) (string, error) {
 	glog.Infof("Saved App Gateway config to %s", tempFile.Name())
 	return tempFile.Name(), nil
 }
+
+func GetHashCode(i interface{}) string {
+	arrBytes := []byte{}
+	jsonBytes, _ := json.Marshal(i)
+	arrBytes = append(arrBytes, jsonBytes...)
```

Comment:
can't we directly use `jsonBytes` instead of creating `arrBytes`

Reply:
Done